security/bug-bounty.md

```diff
@@ -32,6 +32,8 @@ In general, the following would not meet the threshold for severity:
 * Vulnerabilities affecting users of unsupported or unpatched browsers and platforms.
 * Missing best practices in SSL/TLS configuration.
 * Vulnerabilities on sites hosted by third-parties unless they lead to a vulnerability on the Clever services.
+* Vulnerabilities using public API keys for Launchdarkly and other data tracking (e.g. Segment) are NOT in scope, and will get you disqualified from program.
```

Comment:
This actually sounds like the above portion.

We should probably do an explicit callout here:

```suggestion
  * This includes denial of service or causing annoyance on third-parties used by Clever. Actions falling underneath these services will get you disqualified from the program.
```